definitions/tools/hla_consensus.wdl

```diff
@@ -182,7 +182,9 @@ task hlaConsensus {
 
     #Create an exact copy of optitype calls, to be bundled with other relevant
     #files for convenience/later review. Always generated,
-    with open("hla_calls/optitype_calls.txt", "w") as o_c:
+    #NOTE(Layth): this hla_consensus step actually gets passed results from
+    #Optitype for class I and PHLAT for class II. hla_calls will include both results.
```

Comment:
It might be nice to follow up on this tool and fix some of the existing comments (and the variable name?) to reflect its true contents.